clang/runtime/dpct-rt/include/dpct/math.hpp

```diff
@@ -1321,6 +1321,41 @@ inline constexpr RetT extend_max_sat(AT a, BT b, CT c,
   return detail::extend_binary<RetT, true>(a, b, c, maximum(), second_op);
 }
 
+/// Extend \p a and \p b to 33 bit and compare input values using specified
+/// comparison \p cmp .
+///
```

Comment:
Need a empty comment line?

Reply:
Without this blank line, clang-format will output a incorrect format.
https://github.com/oneapi-src/SYCLomatic/blob/ba127eb6ed0c83348ff0330e6ddac3ab71b6170d/clang/include/clang/Analysis/CloneDetection.h#L49-L62